airbyte-integrations/connectors/source-looker/integration_tests/configured_catalog.json

```diff
@@ -3,372 +3,11882 @@
     {
       "stream": {
         "name": "color_collections",
-        "json_schema": {},
-        "supported_sync_modes": ["full_refresh"],
-        "source_defined_cursor": false
+        "json_schema": {
```

Comment:
I think we no longer need to write out the `json_schema` in the `configured_catalog.json` file.

Removing these schemas could make the PR much cleaner. Also we don't need to update the code in two places if there is any change in the schema.

Reply:
these files were generated by my separate script. I've fixed. Now they should include empty json_schemas only.